### ISSUE TITLE
[3.0] External video - Video player doesn't start at the beginning of the video

### ISSUE THREAD

**Issue by antonbsa** [bigbluebutton/bigbluebutton#19245]
**Describe the bug**
The first attempt to share a video is starting in the middle of it

**Video**

https://github.com/bigbluebutton/bigbluebutton/assets/56703993/a41d47f5-ed67-4723-80f0-06863989e38c

**Comment by Wladimir-N** [comment]
The link you paste contains a timecode.

**Comment by antonbsa** [comment]
> The link you paste contains a timecode.

Indeed! good catch hehehe. I have no idea how I did it. however, I just tried getting the link from the search list and I had the same behavior (looks like there's no timecode parameter):

https://github.com/bigbluebutton/bigbluebutton/assets/56703993/1f1d4e94-8af6-4e3b-9b97-ff2352c32e51

notice that in the second attempt, the video is quickly set to 1:18 and returns to the beginning. same link and meeting.

video link: `https://www.youtube.com/watch?v=i1YLcqMd1qo&pp=ygUIZGVlcCBlbmQ%3D`
